test/RelativeStreamTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       http://github.com/zendframework/zend-diactoros for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-diactoros/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Diactoros;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Diactoros\RelativeStream;
+use Zend\Diactoros\Stream;
+
+/**
+ * @covers \Zend\Diactoros\RelativeStream
+ */
+class RelativeStreamTest extends TestCase
+{
+    public function testToString()
+    {
+        $decorated = $this->prophesize(Stream::class);
```

Comment:
I'll actually say that I don't like neither mockery nor prophecy mainly due to the lack of autocompletion. Yes, feel free to throw stuff at me now :-P